spec/content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/__methods__/finish_spec.rb

```diff
@@ -9,17 +9,18 @@
   let(:task) { FactoryGirl.create(:service_template_provision_task, :destination => service_ansible_tower, :miq_request => request) }
   let(:svc_task) { MiqAeMethodService::MiqAeServiceServiceTemplateProvisionTask.find(task.id) }
   let(:svc_service) { MiqAeMethodService::MiqAeServiceServiceAnsibleTower.find(service_ansible_tower.id) }
-  let(:root_object) { Spec::Support::MiqAeMockObject.new('service_template_provision_task' => svc_task) }
+  let(:root_object) { Spec::Support::MiqAeMockObject.new('service' => svc_service, 'service_action' => 'Provision') }
   let(:ae_service) { Spec::Support::MiqAeMockService.new(root_object) }
 
   context "finish" do
     it "check success" do
       allow(svc_task).to receive(:destination).and_return(svc_service)
       allow(svc_service).to receive(:main).and_return(nil)
+      allow(ae_service).to receive(:create_notification)
 
       described_class.new(ae_service).main
 
-      expect(ae_service.root['ae_result']).to eq("ok")
+      expect(ae_service.root['ae_result']).to be_nil
```

Comment:
@tinaafitz 
we are checking the value to be nil but no one is setting it. A better check would be to set the ae_result to ok and check for that.